docs/release-notes/.FSharp.Compiler.Service/9.0.200.md

```diff
@@ -51,6 +51,7 @@
 * Better ranges for #nowarn error reporting; bring back #nowarn warnings for --langVersion:80; add warnings under feature flag ([PR #17871](https://github.com/dotnet/fsharp/pull/17871))
 * CheckAndThrow can be invoked only from within Cancellable context ([PR #18037](https://github.com/dotnet/fsharp/pull/18037))
 * Make ILTypeDef base type calculation lazy. ([PR #18005](https://github.com/dotnet/fsharp/pull/18005))
+* "#if" directive around nullness removed from src/Compiler/TypedTree/TypedTreePickle.fs and refactored. ([PR #18203](https://github.com/dotnet/fsharp/pull/18203))
```

Comment:
Once a first PR from this category gets merged in, you can keep a single changelog entry for the full block of PRs and just append PR numbers into the list, like:

* Nulless #if directives removed from the codebase (PR 1, PR 2, PR 3, ..)